docs/modules/ROOT/pages/how-tos/install/cloudscale.adoc

```diff
@@ -0,0 +1,204 @@
+= Installation on Cloudscale
+
+[abstract]
+--
+Steps to install an OpenShift 4 cluster on https://cloudscale.ch[Cloudscale].
+
+These steps follow the https://docs.openshift.com/container-platform/latest/installing/installing_bare_metal/installing-bare-metal.html[Installing a cluster on bare metal] docs to set up a user provisioned installation (UPI).
+https://www.terraform.io[Terraform] is used to provision the cloud infrastructure.
+--
+
+== Prerequisites
+* Cloudscale API token
+* `terraform`
```

Comment:
We might need to mention the Terraform version, either here or in the Terraform repository.

Reply:
It is already enforced here: https://github.com/appuio/terraform-openshift4-cloudscale/blob/init/provider.tf#L2